### ISSUE TITLE
Undefined property: SnappyMail\HTTP\CSP::$script in .../apps/snappymail/lib/ContentSecurityPolicy.php on line 19

### ISSUE THREAD

**Issue by cscholz** [the-djmaze/snappymail#997]
**Describe the bug**
This one of the last SnappyMail Updates the App can not be access by NC user.

**Error Message logged**
```
"PHP message: PHP Warning:  Undefined property: SnappyMail\HTTP\CSP::$script in /.../apps/snappymail/lib/ContentSecurityPolicy.php on line 19"

{"reqId":"EsiP9nqn3N7YuSfeGEPS","level":3,"time":"March 01, 2023 22:52:32","remoteAddr":"92.216.152.188","user":"cscholz","app":"index","method":"GET","url":"/apps/snappymail/","message":"{\"Exception\":\"Exception\",\"Message\":\"array_merge(): Argument #2 must be of type array, null given in file '/.../apps/snappymail/lib/ContentSecurityPolicy.php' line 19\",\"Code\":0,\"Trace\":[{\"file\":\"/.../lib/private/AppFramework/App.php\",\"line\":172,\"function\":\"dispatch\",\"class\":\"OC\\\\AppFramework\\\\Http\\\\Dispatcher\",\"type\":\"->\",\"args\":[[\"OCA\\\\SnappyMail\\\\Controller\\\\PageController\"],\"index\"]},{\"file\":\"/.../lib/private/Route/Router.php\",\"line\":298,\"function\":\"main\",\"class\":\"OC\\\\AppFramework\\\\App\",\"type\":\"::\",\"args\":[\"PageController\",\"index\",[\"OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer\"],[\"snappymail.page.index\"]]},{\"file\":\"/.../lib/base.php\",\"line\":1047,\"function\":\"match\",\"class\":\"OC\\\\Route\\\\Router\",\"type\":\"->\",\"args\":[\"/apps/snappymail/\"]},{\"file\":\"/.../index.php\",\"line\":36,\"function\":\"handleRequest\",\"class\":\"OC\",\"type\":\"::\",\"args\":[]}],\"File\":\"/.../lib/private/AppFramework/Http/Dispatcher.php\",\"Line\":165,\"Previous\":{\"Exception\":\"TypeError\",\"Message\":\"array_merge(): Argument #2 must be of type array, null given\",\"Code\":0,\"Trace\":[{\"file\":\"/.../apps/snappymail/lib/ContentSecurityPolicy.php\",\"line\":19,\"function\":\"array_merge\",\"args\":[[\"'self'\"],null]},{\"file\":\"/.../apps/snappymail/lib/Controller/PageController.php\",\"line\":39,\"function\":\"__construct\",\"class\":\"OCA\\\\SnappyMail\\\\ContentSecurityPolicy\",\"type\":\"->\",\"args\":[]},{\"file\":\"/.../lib/private/AppFramework/Http/Dispatcher.php\",\"line\":225,\"function\":\"index\",\"class\":\"OCA\\\\SnappyMail\\\\Controller\\\\PageController\",\"type\":\"->\",\"args\":[]},{\"file\":\"/.../lib/private/AppFramework/Http/Dispatcher.php\",\"line\":133,\"function\":\"executeController\",\"class\":\"OC\\\\AppFramework\\\\Http\\\\Dispatcher\",\"type\":\"->\",\"args\":[[\"OCA\\\\SnappyMail\\\\Controller\\\\PageController\"],\"index\"]},{\"file\":\"/.../lib/private/AppFramework/App.php\",\"line\":172,\"function\":\"dispatch\",\"class\":\"OC\\\\AppFramework\\\\Http\\\\Dispatcher\",\"type\":\"->\",\"args\":[[\"OCA\\\\SnappyMail\\\\Controller\\\\PageController\"],\"index\"]},{\"file\":\"/.../lib/private/Route/Router.php\",\"line\":298,\"function\":\"main\",\"class\":\"OC\\\\AppFramework\\\\App\",\"type\":\"::\",\"args\":[\"PageController\",\"index\",[\"OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer\"],[\"snappymail.page.index\"]]},{\"file\":\"/.../lib/base.php\",\"line\":1047,\"function\":\"match\",\"class\":\"OC\\\\Route\\\\Router\",\"type\":\"->\",\"args\":[\"/apps/snappymail/\"]},{\"file\":\"/.../index.php\",\"line\":36,\"function\":\"handleRequest\",\"class\":\"OC\",\"type\":\"::\",\"args\":[]}],\"File\":\"/.../apps/snappymail/lib/ContentSecurityPolicy.php\",\"Line\":19},\"CustomMessage\":\"--\"}","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.57","version":"25.0.4.1"}
```

**Please complete the following information:**
- Browser: Windows 11 Edge, Version 110.0.1587.57
- PHP version: php8.1-fpm
- SnappyMail Version: 2.26.4

**Comment by klermor** [comment]
Same problem [snappymail v 2.26.4 (with Nextcloud) ]
[PHP] Erreur: Undefined property: SnappyMail\HTTP\CSP::$script at /var/www/xxxxxxx/web/apps/snappymail/lib/ContentSecurityPolicy.php#19
GET /apps/snappymail/
[index] Erreur: Exception: array_merge(): Argument #2 must be of type array, null given in file '/var/www/xxxxxxx/web/apps/snappymail/lib/ContentSecurityPolicy.php' line 19 at <<closure>>
GET /apps/snappymail/

**Comment by the-djmaze** [comment]
Already fixed in #991
